### PR TITLE
Clean Code

### PR DESCRIPTION
**Pull request by ouertani**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

Replace l with L
remove optional ;
used some java 8 shortcuts 
### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._
Run tests
### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksql-engine/src/test/java/io/confluent/ksql/planner/plan/KsqlBareOutputNodeTest.java [hunk 1]
```diff
@@ -121,9 +121,9 @@ public void shouldComputeQueryIdCorrectly() {
     // When:
     final Set<QueryId> ids = IntStream.range(0, 100)
         .mapToObj(i -> node.getQueryId(queryIdGenerator))
-        .collect(Collectors.toSet());;
+        .collect(Collectors.toSet());
 
-    // Then:
+      // Then:
```
Comment (agavra) [inline ksql-engine/src/test/java/io/confluent/ksql/planner/plan/KsqlBareOutputNodeTest.java]:
```suggestion
    // Then:
```

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/LongTopkKudafTest.java [hunk 2]
```diff
@@ -30,8 +30,8 @@
 @SuppressWarnings("unchecked")
 public class LongTopkKudafTest {
   private final List<Long> valuesArray = ImmutableList.of(10L, 30L, 10L, 45L, 50L, 60L, 20L, 60L, 80L, 35L,
-      25L);;
-  private TopKAggregateFunctionFactory topKFactory;
+      25L);
+    private TopKAggregateFunctionFactory topKFactory;
```
Comment (spena) [inline ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/LongTopkKudafTest.java]: Nit: incorrect indentation

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/StringTopkKudafTest.java [hunk 3]
```diff
@@ -30,8 +30,8 @@
 @SuppressWarnings("unchecked")
 public class StringTopkKudafTest {
   private final List<String> valueArray = ImmutableList.of("10", "ab", "cde", "efg", "aa", "32", "why", "How are you",
-      "Test", "123", "432");;
-  private TopKAggregateFunctionFactory topKFactory;
+      "Test", "123", "432");
+    private TopKAggregateFunctionFactory topKFactory;
```
Comment (spena) [inline ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/StringTopkKudafTest.java]: Nit: incorrect indentation

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/DoubleTopkDistinctKudafTest.java [hunk 4]
```diff
@@ -27,8 +27,8 @@
 public class DoubleTopkDistinctKudafTest {
 
   private final List<Double> valuesArray = ImmutableList.of(10.0, 30.0, 45.0, 10.0, 50.0, 60.0, 20.0, 60.0,
-      80.0, 35.0, 25.0, 60.0, 80.0);;
-  private final TopkDistinctKudaf<Double> doubleTopkDistinctKudaf
+      80.0, 35.0, 25.0, 60.0, 80.0);
+    private final TopkDistinctKudaf<Double> doubleTopkDistinctKudaf
```
Comment (spena) [inline ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/DoubleTopkDistinctKudafTest.java]: Nit: incorrect indentation

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/StringTopkDistinctKudafTest.java [hunk 5]
```diff
@@ -27,8 +27,8 @@
 public class StringTopkDistinctKudafTest {
 
   private final List<String> valuesArray = ImmutableList.of("10", "30", "45", "10", "50", "60", "20", "60", "80", "35",
-      "25", "60", "80");;
-  private final TopkDistinctKudaf<String> stringTopkDistinctKudaf
+      "25", "60", "80");
+    private final TopkDistinctKudaf<String> stringTopkDistinctKudaf
```
Comment (spena) [inline ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/StringTopkDistinctKudafTest.java]: Nit: incorrect indentation

File: ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java [hunk 6]
```diff
@@ -197,8 +197,8 @@ private PhysicalPlanBuilder buildPhysicalPlanBuilder(
   }
 
   private QueryMetadata buildPhysicalPlan(final String query) {
-    final PlanNode logical = AnalysisTestUtil.buildLogicalPlan(query, metaStore);;
-    return physicalPlanBuilder.buildPhysicalPlan(new LogicalPlanNode(query, logical));
+    final PlanNode logical = AnalysisTestUtil.buildLogicalPlan(query, metaStore);
+      return physicalPlanBuilder.buildPhysicalPlan(new LogicalPlanNode(query, logical));
```
Comment (spena) [inline ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java]: Nit: incorrect indentation

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/DoubleTopkDistinctKudafTest.java (hunk 4 above)
Comment (ouertani) [reply ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/DoubleTopkDistinctKudafTest.java]: resolved

File: ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java (hunk 6 above)
Comment (ouertani) [reply ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java]: resolved

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/StringTopkKudafTest.java (hunk 3 above)
Comment (ouertani) [reply ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/StringTopkKudafTest.java]: resolved

File: ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/StringTopkDistinctKudafTest.java (hunk 5 above)
Comment (ouertani) [reply ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/StringTopkDistinctKudafTest.java]: resolved

**Pull request merged**